api/app/tests/sfms/test_daily_fwi_processor.py

```diff
@@ -62,14 +62,15 @@ async def test_bui_date_range_processor(mocker: MockerFixture):
     get_weather_data_key_spy = mocker.spy(mock_key_addresser, "get_weather_data_keys")
     gdal_prefix_keys_spy = mocker.spy(mock_key_addresser, "gdal_prefix_keys")
     get_calculated_index_key_spy = mocker.spy(mock_key_addresser, "get_calculated_index_key")
-    bui_date_range_processor = BUIDateRangeProcessor(TEST_DATETIME, 2, mock_key_addresser)
+    bui_date_range_processor = DailyFWIProcessor(TEST_DATETIME, 2, mock_key_addresser)
```

Comment:
Would you mind renaming the `bui_date_range_processor` variable and this test case (`test_bui_date_range_processor`)?